Mixpanel/Mixpanel.m

```diff
@@ -191,6 +194,7 @@ - (instancetype)initWithToken:(NSString *)apiToken launchOptions:(NSDictionary *
 
         self.decideResponseCached = NO;
         self.showSurveyOnActive = YES;
+        self.enableABTestDesigner = YES;
```

Comment:
Let's put the product question aside for a moment.

To both of your points, would changing the gesture resolve your concerns?